netbox/templates/dcim/device.html

```diff
@@ -556,6 +556,9 @@ <h1>{{ device }}</h1>
                                 <span class="glyphicon glyphicon-check" aria-hidden="true"></span> Show IPs
                             </button>
                         </div>
+                        <div class="col-md-2 pull-right noprint">
+                            <input class="form-control interface-filter" type="text" placeholder="Filter (RegExp)" style="height: 23px" />
```

Comment:
IMO we should leave the placeholder text as "Filter" since some people won't know what RegExp refers to. It might be nice to add a little question mark icon (`fa-question-circle`) next to the filter field that would pop up a little explanation, but not really necessary.